Source/Hypermedia.Client.Extensions/SystemNetHttp/HttpHypermediaResolver.cs

```diff
@@ -68,23 +71,38 @@ string Quoted(string value)
                 return value;
             }
             HttpResponseMessage response;
+            bool forceRevalidate = ignoreCache;
```

Comment:
With this implementation, the cache is not actually ignored, it just forces a revalidation and as such saves bandwidth in case of a NotModified response.
Would it be more fit to properly ignore the cache here and send a fresh request? The backend might still serve the request from its own cache, which is equivalent to getting a NotModified response.
The use case is when a reload via the Self link is started, to not serve from the local cache, since that is the object that I have already in hand (e.g. the user pressing F5 in the browser).

Reply:
I would name the parameter different: forceRefresh maybe. At this point i should not be aware of a cache (single level of abstraction) and it migth be distracting. The cache should be transparent except when building the client at startup.
I would pass the bool on to  the next level. If there is a cache it will do accordingly else the bool will be ignored (because it will refresh in any case)